modules/apps/transfer/types/transfer_authorization.go

```diff
@@ -145,6 +152,51 @@ func isAllowedAddress(ctx sdk.Context, receiver string, allowedAddrs []string) b
 	return false
 }
 
+// validateMemo returns a nil error indicating if the memo is valid for transfer.
+func validateMemo(ctx sdk.Context, memo string, allowedPacketDataList []string) error {
+	// if the allow list is empty, then the memo must be an empty string
+	if len(allowedPacketDataList) == 0 {
+		if len(strings.TrimSpace(memo)) != 0 {
```

Comment:
should these two conditions be included in a logical AND? 

The current code means you cannot include any memo unless there is something in `allowPacketDataList`. What if I want to add a random string memo "damian's packet" - it would fail.

Reply:
yeah, according to the design if `allowedPacketDataList` is empty, we cannot include any memo. We will allow any memo if the `allowedPacketDataList` is "*"

Reply:
Okay, so authz users must set the `allowedPacketDataList` to use any kind of memo at all. That's good I guess.. definitely explicit 😅